### PR TITLE
RN v0.64 upgrade followups

### PR DESCRIPTION
**Pull request by chrisbobbe**
I think these are all the planned followups, but I could be forgetting one or two. These just remove workarounds that are no longer necessary.

Related: #4426

### CODE REVIEW

File: src/common/LoadingBanner.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { View } from 'react-native';
 import { useSelector } from '../react-redux';
 import { getLoading } from '../selectors';
 // eslint-disable-next-line import/no-useless-path-segments
```
Comment (gnprice) [inline src/common/LoadingBanner.js:9]: This line can probably disappear too, right?

**Review by gnprice — COMMENTED** [review]
Thanks! Looks good, modulo a tiny comment below. Otherwise please merge at will.

File: src/common/LoadingBanner.js (hunk 1 above)
Comment (chrisbobbe) [reply src/common/LoadingBanner.js]: Yep, thanks!

**Comment by chrisbobbe** [comment]
Thanks for the review! Merged.

**Pull request closed without merging**